constants/Marketplaces.ts

```diff
@@ -0,0 +1,6 @@
+export const marketplaces = [
+  { value: 'all', label: 'allMarket' },
+  { value: 'Ebay', label: 'Ebay' },
+  { value: 'Kleinanzeigen', label: 'Kleinanzeigen' },
+  { value: 'AutoScout24', label: 'autoScout' },
```

Comment:
Why is the 24 being removed from the label?

Reply:
It's the translations string

Reply:
and it can't have numbers?

Reply:
Seemed kinda pointless to put numbers in it but I can put them there if you really want them